packages/editor/src/components/visual-editor/index.js

```diff
@@ -341,7 +341,7 @@ function VisualEditor( {
 	const zoomOutProps = isZoomOutMode
 		? {
 				scale: 'default',
-				frameSize: '20px',
+				frameSize: '48px',
```

Comment:
I made the frame size larger so that at small resolutions there is still space for the vertical toolbar: 
<img width="874" alt="Screenshot 2024-06-18 at 14 57 53" src="https://github.com/WordPress/gutenberg/assets/275961/4e511e59-2dd4-49c7-9d0d-be81af719d88">